benchmarks/infra/gpu/stage-1/sample-terraform.tfvars

```diff
@@ -0,0 +1,35 @@
+project_id   = "change-me"
```

Comment:
Hm, shouldn't there be other files in this directory? (infra/gpu/stage-1)

Reply:
@achandrasekar we removed the gpu/ and tpu/ folders in the infra/ section during the refactor.

The necessary files are in stage-1/ and stage-2/

Reply:
I've removed this directory.